package.json

```diff
@@ -33,7 +57,7 @@
     "generate:clickhouse": "rimraf src/autocomplete/databases/clickhouse/generated && antlr4ng -Dlanguage=TypeScript -o src/autocomplete/databases/clickhouse/generated/ -visitor -no-listener -Xexact-output-dir src/autocomplete/databases/clickhouse/grammar/ClickHouseLexer.g4 src/autocomplete/databases/clickhouse/grammar/ClickHouseParser.g4 && bash scripts/patch-generated.sh clickhouse",
     "generate:yql": "rimraf src/autocomplete/databases/yql/generated && antlr4ng -Dlanguage=TypeScript -o src/autocomplete/databases/yql/generated/ -visitor -no-listener -Xexact-output-dir src/autocomplete/databases/yql/grammar/YQL.g4 && bash scripts/patch-generated.sh yql",
     "test": "node --experimental-vm-modules ./node_modules/.bin/jest --verbose src",
-    "build": "rimraf dist && node build.js && tsc -p tsconfig.build.json",
+    "build": "rimraf dist && rimraf dist-types && node build.js && tsc -p tsconfig.build.json --declarationDir dist-types",
```

Comment:
do we need a separate declaration dir? why not just put it in dist?
is this common practice?

Reply:
no, removed dist-types